static/app/views/alerts/rules/issue/details/ruleDetails.spec.jsx

```diff
@@ -197,4 +197,26 @@ describe('AlertRuleDetails', () => {
       )
     ).not.toBeInTheDocument();
   });
+
+  it('renders the mute button and can mute/unmute alerts', async () => {
+    const postRequest = MockApiClient.addMockResponse({
+      url: `/projects/${organization.slug}/${project.slug}/rules/${rule.id}/snooze/`,
+      method: 'POST',
+      data: {target: 'me'},
+    });
+    const deleteRequest = MockApiClient.addMockResponse({
+      url: `/projects/${organization.slug}/${project.slug}/rules/${rule.id}/snooze/`,
+      method: 'DELETE',
+    });
+    createWrapper();
+    expect(await screen.findByText('Mute')).toBeInTheDocument();
+
+    await userEvent.click(screen.getByText('Mute'));
```

Comment:
super nit - but prefer getByRole
```suggestion
    await userEvent.click(screen.getByRole('button', {name: 'Mute'}));
```